tests/test_checkout.py

```diff
@@ -284,8 +284,8 @@ def test(self):
         with open(self.file1_stage, "r") as fd:
             d = yaml.load(fd)
 
-        del (d[Stage.PARAM_OUTS][0][RemoteLOCAL.PARAM_CHECKSUM])
-        del (d[Stage.PARAM_DEPS][0][RemoteLOCAL.PARAM_CHECKSUM])
+        del d[Stage.PARAM_OUTS][0][RemoteLOCAL.PARAM_CHECKSUM]
```

Comment:
👍

Reply:
@shcheklein , would love to hear your input on this one: https://github.com/iterative/dvc/issues/329